core/templates/css/oppia.css

```diff
@@ -2867,6 +2867,111 @@ md-card.preview-conversation-skin-supplemental-card {
   margin-left: 20px;
 }
 
+.oppia-info-card-title {
```

Comment:
Hey @divyanshkhetan, sorry if I'm missing something obvious, but I'm confused as to why these changes are needed here. Could you please explain? Thanks!

Reply:
@divyanshkhetan I think you may have missed the above comment; could you please take a look?

The reason I say I'm confused about these changes is because in [one of the recently merged PRs](https://github.com/oppia/oppia/pull/15047/files), these CSS rules were moved from `oppia.css` to the corresponding template's CSS file (`information-card-modal.component.css`). I'm not sure why these rules need to be added back to `oppia.css`.

Could you please explain this, or let me know if I'm missing something?

Reply:
@nithinrdy No, I moved my changes from `oppia.css` to `information-card-modal.component.css`. After some time, my PR went stale so instead of fetching the changes from github directly, I pulled those changes to my branch on my local machine as I was new to this and then i commited those changes. So, these new changes were part of the upstream and somehow when i pushed these updates in my branch, they reflected as my contribution.

Reply:
Oh, I see.

> I moved my changes from `oppia.css` to `information-card-modal.component.css`

Since you made the required changes in the information-card-modal component, could you delete these additions then? These CSS rules are redundant as they are already present in the actual component's CSS file.

Reply:
Just a heads-up, these CSS rules are already present in the actual corresponding component's CSS file and adding them to oppia.css would be redundant. Please omit these additions from your PR.

Reply:
@nithinrdy Done. Sorry for the delay. Thanks!